pymysql/connections.py

```diff
@@ -472,7 +497,7 @@ def __init__(self, host="localhost", user=None, password="",
                  client_flag=0, cursorclass=Cursor, init_command=None,
                  connect_timeout=None, ssl=None, read_default_group=None,
                  compress=None, named_pipe=None, no_delay=False,
-                 autocommit=False, db=None, passwd=None):
+                 autocommit=False, db=None, passwd=None, load_local=False):
```

Comment:
Could you change option name to `local_infile`?
Options other than DB-API 2.0, I like consistency with MySQL options.
http://dev.mysql.com/doc/refman/5.1/en/server-system-variables.html#sysvar_local_infile